src/scripts/configs/routes.js

```diff
@@ -1,6 +1,7 @@
 import * as SurahsActions from 'actions/SurahsActions';
 import * as AyahsActions from 'actions/AyahsActions';
 import RouteNotFound from 'utils/RouteNotFound';
+import AyahRangeError from 'utils/AyahRangeError';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>'AyahRangeError' is defined but never used.